TESTING.md

```diff
@@ -67,6 +67,12 @@ To run specific versions' backwards compatibility tests, pass the versions to th
 
 `yarn test:bwc -o [test path to opensearch.tar.gz] -d [test path to opensearch-dashboards.tar.gz] -v "[test versions]"`
 
+To generate test data that we be utilized for future version backwards compatibility tests:
```

Comment:
nit - "we be utilized" should be "will be utilized".

It's not entirely clear here what "future version backwards compatibility tests" means. I think the idea is that this command generates test data for the current version as a source of truth that future versions can check against when running BWC.

Reply:
Addressed! Good catch. Let me know if the updated description is more useful.